www-apps/gitea/gitea-1.10.0.ebuild

```diff
@@ -0,0 +1,99 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit golang-vcs-snapshot tmpfiles systemd
+
+EGO_PN="code.gitea.io/gitea"
+
+DESCRIPTION="A painless self-hosted Git service"
+HOMEPAGE="https://gitea.io"
+SRC_URI="https://github.com/go-gitea/gitea/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0 BSD BSD-2 ISC MIT MPL-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64"
+IUSE="+acct pam sqlite"
+
+BDEPEND="dev-lang/go"
+COMMON_DEPEND="
+	acct? (
+		acct-group/git
+		acct-user/git[gitea]
+	)
+	pam? ( sys-libs/pam )"
+DEPEND="${COMMON_DEPEND}"
```

Comment:
Looks like COMMON_DEPEND isnt needed here, since you don't have different deps in DEPEND and RDEPEND. You can put the contents of COMMON_DEPEND into DEPEND, then inherit that in RDEPEND.

Is acct-*/ really needed in build-time? It's usually not and I couldn't find examples where it's needed here either...

Reply:
> Then the customary question: Would you like to maintain this again :)

well, i would do PR's with best effort, however I do not understand the proxy maintaner project. the PR's are not getting merged faster - and upstream is quite fast (version 1.9.[0-4] have been skipped), resulting in an often out-dated package, which looks like doing a bad job as a maintainer -  i don't want to take responsibility for that.

Reply:
> Is acct-*/ really needed in build-time? It's usually not and I couldn't find examples where it's needed here either...

no it is used at install time. so i should put it to `RDEPEND`